packages/react-charting/jest.config.js

```diff
@@ -9,7 +9,7 @@ function getEsmOnlyPackagesToCjsMapping() {
    * map of packages that ship only as ESM
    */
   const cjsPathsToEsmOnlyPackages = {
-    '^d3-scale$': prefix + 'node_modules/d3-scale/dist/d3-scale.js',
+    '^d3-scale$': prefix + '../../node_modules/d3-scale/dist/d3-scale.js',
```

Comment:
can we document this behaviour inline ( I know it's described within PR description ) ?

Reply:
Good point. Added.